crates/flipperzero/src/furi/kernel.rs

```diff
@@ -0,0 +1,89 @@
+//! Furi Kernel primitives.
+
+use flipperzero_sys as sys;
+use ufmt::derive::uDebug;
+
+use crate::furi;
+
+/// Check if CPU is in IRQ; or kernel running and IRQ is masked.
+pub fn is_irq_or_masked() -> bool {
+    unsafe { sys::furi_kernel_is_irq_or_masked() }
+}
+
+/// Check if kernel is running.
+pub fn is_running() -> bool {
+    unsafe { sys::furi_kernel_is_irq_or_masked() }
```

Comment:
Looks like the wrong function is called

Reply:
Good catch! Copy-paste mistake.

Fixed.